frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.scss

```
@@ -0,0 +1,6 @@
+.odc-traffic-splitting-modal {
+  .odc-multi-column-field__col {
+    flex: auto;
+    width: 30%;
```

```suggestion
    flex-grow: 1;
```
Grow to the same size? I haven't tested this out, not sure if the `grow-basis` will impact the solution, may need `grow: 1 1 0;`

---

I tried ```flex-grow``` and it does work in this case as the width of the dropdown varies as per the option length. That is why it is not aligned.

---

Flex-box in the horizontal direction handles width. You don't need the width property. If it's not working you're conflicting with other styles / indeterminate items. I'll look more into it in a few, haven't checked out the code yet.